app/services/index.js

```diff
@@ -1,2 +1,3 @@
-export { default as voteService } from './topics';
+import voteService from './topics';
```

Comment:
`default as voteService` is not a problem, but using at least one export default is recommended in ESLint. [Link](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md)
And for scalability, when other services are added, we need `export default` that exports whole services at once.